neuraxle/base.py

```diff
@@ -1423,6 +1442,18 @@ def fit_transform(self, data_inputs, expected_outputs):
             step.set_train(is_train)
         return self
 
+    def __repr__(self):
+
+        output = self.__class__.__name__ + '\n' \
+                 + "(\n\t" + super(TruncableSteps, self).__repr__() \
+                 + "(\n\t\t" + pprint.pformat(self.steps_as_tuple) \
+                 + "\t\n)" \
+                 + "\n)"
```

Comment:
Something feels wrong here but we'll fix that in another PR later, that's fine for now.